test/specs/WebexMeeting/WebexMeeting.e2e.js

```diff
@@ -1,10 +1,68 @@
-describe('my widget', () => {
+import WebexMeetingPage from './WebexMeetingPage';
+
+describe('Meeting Widget', () => {
+  beforeAll(() => {
+    return WebexMeetingPage.createUser();
+  });
+
+  afterAll(() => {
+    return WebexMeetingPage.removeUser();
+  });
+
   it('should find sections and match text', () => {
-    browser.url('https://webex.github.io/widgets/');
+    WebexMeetingPage.open();
 
     // should contain two sections
     expect($$('.content > section').length).toBe(2);
     // should match first section text
     expect($('.content > section > h3')).toHaveTextContaining('Webex Widgets Access Token');
   });
+
+  it('should mute audio before joining meeting', async () => {
```

Comment:
let's drop the "shoulds", they look like unnecessary boilerplate: `mutes audio before joining session`

https://github.com/CareMessagePlatform/jasmine-styleguide#speak-human